docs/self-managed/zeebe-deployment/operations/backups.md

```diff
@@ -0,0 +1,81 @@
+---
+id: backups
+title: "Backups"
+description: "Overview and Guide for taking Zeebe backups"
+---
+
+Zeebe fully manages the state of your process instances and does not use an external DBMS, you might want to consider taking backups of Zeebe data.
+Zeebe is fault tolerant and replicates state internally.
+Backups are only necessary if you want to protect against the loss of entire replica sets or data corruption bugs.
+
+State of other component, such as Operate and Tasklist, is not managed by Zeebe and needs to be backed up separately.
+
+Taking backups is a manual process that is highly dependent on you infrastructure and deployment.
+We do not provide an automated backup mechanism or tool.
+
+## Cold Backups
+
+Cold backups, also called offline backups, require **downtime**.
+During the downtime, processes don't make progress and clients can't communicate with Zeebe.
+We recommend that you test how your clients behave during the downtime or that you shut down clients as well.
+
+### Shutting down all brokers in you cluster
+To take a consistent backup, all brokers must be shut down first.
+
+As soon as brokers are shutting down, partitions become unhealthy and clients will lose connections to Zeebe or experience full backpressure.
```

Comment:
I was thinking whether this also needs some time to flush everything to disk? I.e. the the shutdown has to be completed?

Reply:
I'll add a sentence to clarify that one should wait for the shutdown to **complete** before taking a snapshot :+1: